src/components/ScanPatientPage/ScanPatientTopBar.tsx

```diff
@@ -52,7 +52,7 @@ const ScanPatientTopBar = () => {
             color="inherit"
             className={classes.enterCodeText}
             component={NavLink}
-            to={`${pathname}/manual`}
+            to={`${pathname.split('scan')[0]}manual`}
```

Comment:
If someone has a better way of changing the routes, please let me know and I will make that change

Reply:
what's the reasoning behind doing it this way? shouldn't `pathname` not have the substring `scan` anymore with your changes to `App.tsx`?

Reply:
No because this button is still on the `/events/:eventID/ccps/:ccpID/scan` route while we need to navigate to `events/:eventID/ccps/:ccpID/manual`